src/OnboardingSPA/components/HeaderSiteGen/index.js

```diff
@@ -0,0 +1,13 @@
+import { memo } from '@wordpress/element';
+
+import StepNavigation from './step-navigation';
+
+const Header = () => {
+	return (
+		<div className="nfd-onboarding-header">
```

Comment:
I think we might need better naming so that the classes don't clash with the existing onboarding flow.